plasmapy/physics/transport/collisions.py

```diff
@@ -934,6 +935,31 @@ def collision_rate_ion_ion(T_i,
         Method used for Coulomb logarithm calculation (see that function
         for more documentation). Choose from "classical" or "GMS-1" to "GMS-6".
 
+    Notes
+    -----
+    Equations (2.36) and (2.122) in [3]_ provide the original source used
+    to implement this formula, however, in our implementation we use the very
+    same process that leads to the fundamental electron collison rate (2.17),
+    gaining simply a different coefficient:
+
+    .. math::
+        \nu_i = \frac{8}{3 * 4 * \sqrt{\pi}} \nu(v_{Ti})
+
+    Where :math:`\nu` is the general collision frequency and :math:`v_{Ti}`
+    is the ion thermal velocity (the average, for a Maxwellian distribution).
+
+    Note that in the derivation, it is assumed that electrons are present
+    in such numbers as to establish quasineutrality, but the effects of the
+    test ions colliding with them are not considered here. This is a very
+    typical approximation in transport theory.
+
+    This result is an ion momentum relaxation rate, and is used in many
+    classical transport expressions. It is equivalent to:
+    * 1/tau_i from ref [1]_ eqn (1) pp. #,
```

Comment:
For the Braginskii article (ref [1]), I believe this should be equation (2.5i) on page 215.  That equation defines tau_i.  It would probably be worth checking the next two equations and page numbers to double check those too.